### PR TITLE
remove husky and lint staged, disable git hooks

### LINKED ISSUE #1814: rework git hooks
- [x] Remove husky + lint-staged in favour of left-hook
- [x] Make left-hook opt-in
- [x] Remove all hooks, only leave prettier before commit

https://github.com/evilmartians/lefthook/wiki/Migration-from-husky-with-lint-staged

=> Hooks will be fast, will never fail and opt-in

### PR DESCRIPTION
**Pull request by vacekj**
fixes #1814 

left in a lefthook to trigger prettier for those who want

### CODE REVIEW

**Review by boudra — CHANGES_REQUESTED** [review]
Since hooks are not installed automatically now, we should add information of how to use lefthook in the README and once we merge we can share the link to the team so they know how to enable it if they want to.

**Comment by melissa-neira** [comment]
defer to eng review 🕺

**Review by gravityblast — DISMISSED** [review]

**Comment by gravityblast** [comment]
I think we should put the prettier check in CI but it's longer, let start with what @boudra said

**Review by DanieleSalatti — APPROVED** [review]

**Pull request merged**
